src/nixos/machines/sinnenfreude/config/disks.nix

```diff
@@ -195,10 +97,14 @@ in {
 
 									content = {
 										type = "btrfs";
-										extraArgs = [ "--label ROOT_NIXOS" ];
+										extraArgs = [ "--label NIX_STORE" ];
 										subvolumes = {
-											"@" = {
-												mountpoint = "/";
+											"@nix" = {
+												mountpoint = "/nix";
+												mountOptions = [ "compress=lzo" "noatime" ];
```

Comment:
You'll also benefit by adding `"nodiratime"`

Reply:
That would disable logging access time, i don't think that's a good idea